src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -588,6 +588,54 @@ public virtual async Task PatchWithNoPrimaryKeyRouteTest()
                     );
         }
 
+        /// <summary>
+        /// Test to validate that PATCH operation on inaccessible row (by virtue of database policy) fails.
```

Comment:
please add more detail to how this is validating.
e.g. trying to patch field name on entity X but the db policy for role policy_tester_REST should restrict this operation because policy `<policy string>` condition is not met.